.github/workflows/license.yml

```diff
@@ -44,4 +44,14 @@ jobs:
 
     - name: Check licenses
       # We allow only "notice" type of licenses.
-      run: go run github.com/google/go-licenses check --ignore=golang.org/x --allowed_licenses=Apache-2.0,Apache-3,BSD-3-Clause,BSD-4-Clause,CC0-1.0,ISC,MIT ./...
+        run: |
+          go run github.com/google/go-licenses check \
+            # Ignore our own codebase.
```

Comment:
Why do you need this?

Reply:
I don't quite follow. Are you checking our modules or not?

I'm asking because we might at some point have some modules in our repository that have other licenses that are not compatible with Apache (e.g. GPL) and we don't want to miss those dependencies in the Apache module if they get added by mistake.